2024-06-27 08:12:57,220 - distributed.nanny - INFO -         Start Nanny at: 'tcp://128.117.208.62:43305'
2024-06-27 08:13:00,933 - distributed.worker - INFO -       Start worker at: tcp://128.117.208.62:43153
2024-06-27 08:13:00,933 - distributed.worker - INFO -          Listening to: tcp://128.117.208.62:43153
2024-06-27 08:13:00,933 - distributed.worker - INFO -           Worker name:               PBSCluster-8
2024-06-27 08:13:00,933 - distributed.worker - INFO -          dashboard at:       128.117.208.62:44769
2024-06-27 08:13:00,933 - distributed.worker - INFO - Waiting to connect to: tcp://128.117.208.86:37311
2024-06-27 08:13:00,933 - distributed.worker - INFO - -------------------------------------------------
2024-06-27 08:13:00,933 - distributed.worker - INFO -               Threads:                          1
2024-06-27 08:13:00,934 - distributed.worker - INFO -                Memory:                   3.73 GiB
2024-06-27 08:13:00,934 - distributed.worker - INFO -       Local Directory: /glade/derecho/scratch/mdfowler/tmp/dask-scratch-space/worker-nj2rqe38
2024-06-27 08:13:00,934 - distributed.worker - INFO - -------------------------------------------------
2024-06-27 08:13:18,567 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-06-27 08:13:18,569 - distributed.worker - INFO -         Registered to: tcp://128.117.208.86:37311
2024-06-27 08:13:18,569 - distributed.worker - INFO - -------------------------------------------------
2024-06-27 08:13:18,570 - distributed.core - INFO - Starting established connection to tcp://128.117.208.86:37311
2024-06-27 09:22:43,669 - distributed.nanny.memory - WARNING - Worker tcp://128.117.208.62:43153 (pid=19921) exceeded 95% memory budget. Restarting...
2024-06-27 09:22:43,695 - distributed.nanny - INFO - Worker process 19921 was killed by signal 15
2024-06-27 09:22:43,700 - distributed.nanny - WARNING - Restarting worker
2024-06-27 09:22:47,603 - distributed.worker - INFO -       Start worker at: tcp://128.117.208.62:45649
2024-06-27 09:22:47,603 - distributed.worker - INFO -          Listening to: tcp://128.117.208.62:45649
2024-06-27 09:22:47,603 - distributed.worker - INFO -           Worker name:               PBSCluster-8
2024-06-27 09:22:47,603 - distributed.worker - INFO -          dashboard at:       128.117.208.62:36981
2024-06-27 09:22:47,603 - distributed.worker - INFO - Waiting to connect to: tcp://128.117.208.86:37311
2024-06-27 09:22:47,603 - distributed.worker - INFO - -------------------------------------------------
2024-06-27 09:22:47,603 - distributed.worker - INFO -               Threads:                          1
2024-06-27 09:22:47,603 - distributed.worker - INFO -                Memory:                   3.73 GiB
2024-06-27 09:22:47,604 - distributed.worker - INFO -       Local Directory: /glade/derecho/scratch/mdfowler/tmp/dask-scratch-space/worker-bmx473mo
2024-06-27 09:22:47,604 - distributed.worker - INFO - -------------------------------------------------
2024-06-27 09:22:50,944 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-06-27 09:22:50,946 - distributed.worker - INFO -         Registered to: tcp://128.117.208.86:37311
2024-06-27 09:22:50,946 - distributed.worker - INFO - -------------------------------------------------
2024-06-27 09:22:50,947 - distributed.core - INFO - Starting established connection to tcp://128.117.208.86:37311
2024-06-27 10:54:35,975 - distributed.worker - ERROR - Worker stream died during communication: tcp://128.117.208.106:45739
ConnectionRefusedError: [Errno 111] Connection refused

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/glade/u/apps/opt/conda/envs/npl-2024a/lib/python3.11/site-packages/distributed/comm/core.py", line 342, in connect
    comm = await wait_for(
           ^^^^^^^^^^^^^^^
  File "/glade/u/apps/opt/conda/envs/npl-2024a/lib/python3.11/site-packages/distributed/utils.py", line 1935, in wait_for
    return await fut
           ^^^^^^^^^
  File "/glade/u/apps/opt/conda/envs/npl-2024a/lib/python3.11/site-packages/distributed/comm/tcp.py", line 560, in connect
    convert_stream_closed_error(self, e)
  File "/glade/u/apps/opt/conda/envs/npl-2024a/lib/python3.11/site-packages/distributed/comm/tcp.py", line 140, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc.__class__.__name__}: {exc}") from exc
distributed.comm.core.CommClosedError: in <distributed.comm.tcp.TCPConnector object at 0x14b34a0ea090>: ConnectionRefusedError: [Errno 111] Connection refused

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/glade/u/apps/opt/conda/envs/npl-2024a/lib/python3.11/site-packages/distributed/worker.py", line 2056, in gather_dep
    response = await get_data_from_worker(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/glade/u/apps/opt/conda/envs/npl-2024a/lib/python3.11/site-packages/distributed/worker.py", line 2857, in get_data_from_worker
    comm = await rpc.connect(worker)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/glade/u/apps/opt/conda/envs/npl-2024a/lib/python3.11/site-packages/distributed/core.py", line 1619, in connect
    return await self._connect(addr=addr, timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/glade/u/apps/opt/conda/envs/npl-2024a/lib/python3.11/site-packages/distributed/core.py", line 1563, in _connect
    comm = await connect(
           ^^^^^^^^^^^^^^
  File "/glade/u/apps/opt/conda/envs/npl-2024a/lib/python3.11/site-packages/distributed/comm/core.py", line 368, in connect
    raise OSError(
OSError: Timed out trying to connect to tcp://128.117.208.106:45739 after 30 s
2024-06-27 10:55:47,906 - distributed.core - INFO - Connection to tcp://128.117.208.86:37311 has been closed.
2024-06-27 10:55:47,908 - distributed.worker - INFO - Stopping worker at tcp://128.117.208.62:45649. Reason: worker-handle-scheduler-connection-broken
2024-06-27 10:55:47,918 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://128.117.208.62:43305'. Reason: worker-handle-scheduler-connection-broken
2024-06-27 10:55:47,948 - distributed.nanny - INFO - Worker closed
2024-06-27 10:55:50,645 - distributed.nanny - INFO - Closing Nanny at 'tcp://128.117.208.62:43305'. Reason: nanny-close-gracefully
2024-06-27 10:55:50,646 - distributed.dask_worker - INFO - End worker
